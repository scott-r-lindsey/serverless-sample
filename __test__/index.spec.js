'use strict';

const {handler} = require('../src/index');
const {eventStub} = require('./stubs/eventHttpApiGateway.json');

describe('Sample Code Test', () => {

  beforeAll(() => {
    // we can do some setup here
  });

  afterEach(() => {
    // cleanup
  });


  test('True is True', () => {
    expect(true).toBeTruthy();
  });

  test('Returns JSON with expected values', () => {
    const event = eventStub;
    const context = {};

    const result = handler(event, context);
    result.then(data => expect(data).toMatchSnapshot());
  });


});



