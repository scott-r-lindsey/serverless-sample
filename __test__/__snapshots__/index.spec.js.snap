// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Code Test Returns JSON with expected values 1`] = `
Object {
  "body": "{
  \\"message\\": \\"Hello world!\\",
  \\"details\\": \\"The sample code executed correctly xyz\\"
}",
  "statusCode": 200,
}
`;
